contracts/Remittance.sol

```diff
@@ -91,4 +92,15 @@ contract Remittance is Stoppable {
         
         return true;
     }
+    
+    function kill() _onlyOwner public returns(bool success) {
+        if (isKillingProcessStarted()) {
```

Comment:
The purpose of killing a contract is to end it at any moment, and if someone is killing a contract; it means something bad happened, imagine some hacker found a way to steal all Ether from your contract and you can't kill the contract because the deadline is not reached!!! Not good 😉